examples/docker/kafka-oauth-strimzi/compose-authz.yml

```diff
@@ -11,14 +11,14 @@ services:
       - 9092:9092
 
       # javaagent debug port
-      - 5006:5006
+      #- 5006:5006
 
     environment:
 
       # Java Debug
-      KAFKA_DEBUG: y
-      DEBUG_SUSPEND_FLAG: y
-      JAVA_DEBUG_PORT: 5006
+      #KAFKA_DEBUG: y
+      #DEBUG_SUSPEND_FLAG: y
+      #JAVA_DEBUG_PORT: 5006
```

Comment:
maybe we need a comment like "uncomment following lines for enabling debugging ... "

Reply:
I think it should be obvious to everyone that removing hashes would activate these settings.
And I prefer to not add comments for obvious things.